test/kwtool/crawlers_test.exs

```diff
@@ -67,6 +67,30 @@ defmodule Kwtool.CrawlersTest do
                per_page: 15
              } == pagination
     end
+
+    test "returns a list of matched keywords when given a query params" do
```

Comment:
Is it supposed to be 🤔 

1)
```suggestion
    test "returns a list of matched keywords when given a matched query params" do
```

or

2)
```suggestion
    test "returns a list of matched keywords when given a query param" do
```
If it's 2, probably remove -s on line #85 as well

Reply:
i updated follow the (2) approach! Thanks 👍,